pglogrepl.go

```diff
@@ -283,6 +283,230 @@ func StartReplication(ctx context.Context, conn *pgconn.PgConn, slotName string,
 	}
 }
 
+type BaseBackupOptions struct {
+	// Request information required to generate a progress report, but might as such have a negative impact on the performance.
+	Progress          bool
+	// Sets the label of the backup. If none is specified, a backup label of 'wal-g' will be used.
+	Label             string
+	// Request a fast checkpoint.
+	Fast              bool
+	// Include the necessary WAL segments in the backup. This will include all the files between start and stop backup in the pg_wal directory of the base directory tar file.
+	Wal               bool
+	// By default, the backup will wait until the last required WAL segment has been archived, or emit a warning if log archiving is not enabled.
+	// Specifying NOWAIT disables both the waiting and the warning, leaving the client responsible for ensuring the required log is available.
+	NoWait            bool
+	// Limit (throttle) the maximum amount of data transferred from server to client per unit of time (kb/s).
+	MaxRate           int
+	// Include information about symbolic links present in the directory pg_tblspc in a file named tablespace_map.
+	TablespaceMap     bool
+	// Disable checksums being verified during a base backup.
+	// Note that NoVerifyChecksums=true is only supported since PG11
+	NoVerifyChecksums bool
+}
+
+func (bbo BaseBackupOptions) Sql() string {
```

Comment:
`Sql` -> `SQL`

Reply:
Yeah, but it could be private too, so I changed it to sql.